data/resources-config.json

```diff
@@ -1275,6 +1275,21 @@
         "zshrc"
       ]
     },
+    {
+      "type": "github",
+      "username": "SpaceVim",
+      "repo": "SpaceVim",
+      "tags": [
+        "spacevim",
+        "distribution",
+        "neovim-config",
+        "neovim-configuration",
+        "neovim-plugin",
```

Comment:
Please review the available tags and find the ones that match.

Reply:
I have updated the tags of spacevim, thanks.